cmd/res/provision_watchers/llrp.provision.watcher.yaml

```diff
@@ -1,11 +1,11 @@
 name: "llrp-provision-watcher"
-identifiers: 
+serviceName: "device-rfid-llrp"
+identifiers:
   vendorPEN: "."
-  
-blockingidentifiers: 
-  vendorPEN: "25882"
+blockingIdentifiers:
+  vendorPEN:
+    - "25882"
```

Comment:
Is this a new thing, or something broken in the conversion from json to yaml?

Reply:
The way YAML handles map[string][]string